sensor/upgrader/upgradectx/context.go

```diff
@@ -345,3 +352,8 @@ func (c *UpgradeContext) ListCurrentObjects() ([]*unstructured.Unstructured, err
 
 	return objects, nil
 }
+
+// IsPodSecurityEnabled returns whether or not pod security polices are enabled for this cluster
+func (c *UpgradeContext) IsPodSecurityEnabled() bool {
+	return c.podSecurityPoliciesSupported
```

Comment:
@porridge I shared the wrong PR with you.  Moving your comment to this one.
porridge 

This field seems to be set based on OrderedBundleResourceTypes which according to the comment will contain PSPs for all eternity, so I cannot see how this function could ever return anything else than true?

Reply:
Right, I wonder why I didn't see the `if _, ok := resourceMap[gvk]; ok` line. This makes sense.

I have a different question then - why are we special-casing PSPs? In theory in future releases k8s can drop support for any of the resources we mention, right? :thinking:

Reply:
That is a very fair question and a valid point.  One I will probably leave for the code owners to resolve in a less time constrained manner.